include/QuartzCore/CAShapeLayer.h

```diff
@@ -34,8 +34,8 @@ CA_EXPORT_CLASS
 
 @property CGPathRef path;
 @property CGColorRef fillColor;
-@property (copy) NSString* fillRule STUB_PROPERTY;
-@property (copy) NSString* lineCap STUB_PROPERTY;
+@property (copy) NSString* fillRule;
```

Comment:
It doesn't; however, there's no `the object you pass in will be ignored completely, as objects go` annotation; assign would be truer, but copy is what the reference platform specifies and it is expected for all strings.

Since we've implemented it in terms of an enumerated value, we hold true to one of the core tenets of `copy`: The string they pass into the setter and the string they get back from the getter *may* not be the same pointer ;)